plugin-server/src/utils/db/db.ts

```diff
@@ -662,7 +675,12 @@ export class DB {
                         // `addDistinctIdPooled`
                         (_, index) => `, distinct_id_${index} AS (
                         INSERT INTO posthog_persondistinctid (distinct_id, person_id, team_id, version)
-                        VALUES ($${10 + index}, (SELECT id FROM inserted_person), $5, $9))`
+                        VALUES (
+                            $${11 + index + distinctIdVersions!.length - 1},
```

Comment:
Kind of surprised that the type checker can't just infer that `distinctIdVersions` can't be `undefined` at this point.

Reply:
I was too, vscode gave me no signal that it was made but for some reason the `tsc` build itself failed without this...